docs/products/flink/concepts/custom-jars.rst

```diff
@@ -0,0 +1,31 @@
+Custom JARs in Aiven for Apache Flink®
+=================================================
+
+Aiven for Apache Flink® allows you to upload, deploy, and manage your own Java code with custom JARs. This feature aims to expand the capabilities of Aiven for Apache Flink, allowing you to add custom functions that extend beyond the typical SQL-based features. With Custom JARs, you can swiftly develop and maximize the potential of your Aiven for Flink application.
```

Comment:
do we need to say Aiven For Flink application here? It's clumsy. Could we just say Flink application?

Reply:
I agree! But if I recall correctly, this was done for trademark reasons and to differentiate between the Flink application (open-source) and Aiven for Flink applications.